### ISSUE TITLE
chip: button sizes should be WCAG 2.2 2.5.8 AA compliant

### ISSUE THREAD

**Issue by nikkimk** [patternfly/patternfly-elements#2662]
## Description of the issue

Button sizes on chip and chip group should be a 24px minimum, or  [WCAG 2.2 2.5.8 AA compliant](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#258-target-size-minimum-aa).

### Impacted component(s)

- `pf-chip`
- `pf-chip-group`

### Steps to reproduce

1. Go to [chip documentation](https://patternflyelements.org/components/chip/)
2. Inspect the the close button on a chip.
3. Inspect the the close button on a chip group.
4. Inspect the the overflow button on a chip group.

### Expected behavior

Minimum width and height should be 24px.

**Comment by bennypowers** [comment]
we can't solve this here, as we are **beholden to upstream's design choices**

we can file an issue on patternfly, but we have to ship this as is.
